app/players/dwarves/dwarf.py

```diff
@@ -0,0 +1,13 @@
+from abc import ABC
+
```

Comment:
This is redundant empty string, it is better to delete it.

Reply:
Fixed